dist/gmusic-theme.js

```diff
@@ -1008,155 +1013,155 @@ Converter.prototype.getValues = function(space) {
 
 module.exports = convert;
 },{"./conversions":3}],5:[function(require,module,exports){
-module.exports = {
```

Comment:
Just spotted something interesting. Ran it on a vanilla clone of master, and in gmusic-theme.js it's stripped out `\r\n` and replaced it with `\n'

Did you do you last build on Windows by any chance?